check/__processes__/processes.go

```diff
@@ -0,0 +1,113 @@
+package __processes__
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"strings"
+
+	"github.com/adyatlov/bun"
+)
+
+const (
+	name		= "__processes__"
```

Comment:
Thanks for the contribution! Do you mind changing the name of the check to make it simpler for users to launch the check from the command line:
```
bun check __processes__
```
Looks a bit unusual.